pkg/controller/servicebinding/flag.go

```diff
@@ -0,0 +1,17 @@
+package servicebinding
```

Comment:
I would not introduce the package just for this.. please move this file under `cmd/manager`.